src/opendkim.cc

```diff
@@ -407,6 +408,19 @@ NAN_METHOD(OpenDKIM::SigGetError) {
   info.GetReturnValue().Set(Nan::New<v8::Int32>(data));
 }
 
+NAN_METHOD(OpenDKIM::SigGetErrorStr) {                                          
+  const char *errorstr = NULL;                                                  
+                                                                                
+  if (info.Length() != 1) {                                                     
+    Nan::ThrowTypeError("sig_geterrorstr(): Wrong number of arguments");        
+    return;                                                                     
+  }                                                                             
+                                                                                
+  errorstr = dkim_sig_geterrorstr(info[0]->NumberValue());                                               
+                                                                                
+  info.GetReturnValue().Set(Nan::New<v8::String>(errorstr).ToLocalChecked());   
+}
+
```

Comment:
@desttinghim I made this a little cleaner.  less lines of code, and I got rid of the memory allocation.  If you dig down into `dkim_sig_geterrorstr()` you can see the returned value is just a `const char *` from a mapping table.  No need to get the length, allocate more memory, copy it, and then free it, we can just use that string to pass to our new `v8::String`.  Let me know if you have questions.